README.md

```diff
@@ -37,7 +37,7 @@ For more information on Data API, see its <a href="https://developer.here.com/do
 
 When new API is introduced in Data SDK for C++, the old one is not deleted straight away. The standard API deprecation time is 6 months. It gives you time to switch to new code. However, we do not provide ABI backward compatibility.
 
-All of the deprecated methods, functions, and parameters are documented in the Data SDK for C++ <a href="https://developer.here.com/documentation/sdk-cpp/api_reference/index.html" target="_blank">API Reference</a> and <a href="https://github.com/heremaps/here-data-sdk-cpp/blob/master/CHANGELOG.md" target="_blank">changelog</a>.
+Learn more about deprecated methods, functions, and parameters in <a href="https://github.com/heremaps/here-data-sdk-cpp/blob/master/CHANGELOG.md" target="_blank">changelog</a>.
```

Comment:
Why did you remove the link to the API reference?
It is where users can check whether the needed methods, functions, and parameters are deprecated.

Reply:
The API Reference, which I deleted, leads us to the following link > https://developer.here.com/documentation/sdk-cpp/1.15.0/api_reference/index.html (developer.here.com > HERE Data SDK for C++ > API Reference C++. 

Now, we are in the **README.md** file, which contains same content (=sections) as in the **API Reference for C++** on developer.here.com ((https://developer.here.com/documentation/sdk-cpp/1.15.0/api_reference/index.html):
2. Test coverage
3. Backward compatibility
4. Supported platforms
5. Dependencies
6. Use the SDK
7. Contribution
8. License

Are those files different? 
I mean the **README.md** and the file on **https://developer.here.com/documentation/sdk-cpp/1.15.0/api_reference/index.html**?

Reply:
Discussed. Moved the "in the Data SDK for C++ <a href="https://developer.here.com/documentation/sdk-cpp/api_reference/index.html" target="_blank">API Reference</a>" link back.